rotkehlchen/rotkehlchen.py

```diff
@@ -885,6 +891,7 @@ def _query_and_populate_exchange_asset_movements(
             only_cache: bool,
     ) -> List[AssetMovement]:
         """Queryies exchange for asset movements and adds it to all_movements"""
```

Comment:
There is a typo there. (from before)
```suggestion
        """Queries exchange for asset movements and adds it to all_movements"""
```